### PR TITLE
ARTIC section update

### PR DESCRIPTION
**Pull request by wm75**

### CODE REVIEW

**Comment by wm75** [comment]
@pvanheus I've extended the WF quite a bit and made it more similar to the ones from the genomics section.
Hope you like it!

**Comment by wm75** [comment]
@bgruening maybe add an EU logo?

**Comment by bgruening** [comment]
Awesome. I think we can add a logo later. This information is great and should go in asap. @Slugger70 you might want to rerun your data ontop of this workflow. Wolfgang any example history that we can attach?

File: artic/README.md [hunk 1]
```diff
@@ -7,43 +7,92 @@ title: Artic
...
+2. A BED file describing the binding sites of all primers used to generate the tiled amplicons
 
-Note that the V3 BED file has a [known](https://github.com/artic-network/artic-ncov2019/issues/12) formatting
-problem which can be solved with a command line command (`awk 'BEGIN {OFS="\t"; FS="\t"} $5=60' ARTICv3.bed  >ARTICv3_fixed.bed`)
-or the equivalent awk program (`BEGIN {OFS="\t"; FS="\t"} $5=60}`) run in the Galaxy awk tool.
+   The ARTIC project provides BED files for use with this workflow as part of their [SARS-CoV-2 resources repo](https://github.com/artic-network/artic-ncov2019/tree/master/primer_schemes/nCoV-2019).
+   
+   For reproducibility we provide here copies of the:
+   
+   - [v3 primer scheme BED file](nCoV-2019_v1.bed)
```
Comment (Slugger70) [inline artic/README.md:75]: The v3 and v1 links point to the same file

**Comment by Slugger70** [comment]
No worries. I'll start it running today.

File: artic/README.md (hunk 1 above)
Comment (wm75) [reply artic/README.md]: Tahnks for catching @Slugger70!

**Comment by wm75** [comment]
@bgruening I'll add a history link later (as soon as I have one). Currently I've only tested this on in-house data.

**Comment by Slugger70** [comment]
Hey @wm75, what's the zip collection step for? It doesn't go anywhere? Would it be better to take the output of that and send it down the line rather than the individual readsets?

**Comment by wm75** [comment]
That's part of our workflow scheduling workaround:
You don't want to pass list:paired collections to tools, but if the user passes them in, then the fastp result should take the same form. Kind of cosmetic, and certainly can be removed.

**Pull request merged**